lib/input_validator.rb

```diff
@@ -0,0 +1,25 @@
+require_relative "board"
```

Comment:
Why's this require here?

Reply:
I thought I needed that for the get_valid_moves method, but the tests are passing without it. I'll delete that line.